analysis/active_analyses.R

```diff
@@ -562,21 +564,23 @@ for (c in cohorts) {
                          episode_event_threshold = episode_event_threshold,
                          covariate_threshold = covariate_threshold,
                          age_spline = TRUE,
-                         analysis = "sub_prior_parkinson_false")
-    }
-  
-  for (i in parkinson_disease) { #remove cov_bin_history_parkinson
-    
-    ## analysis: sub_prior_parkinson_risk_true ---------------------------------------
-    
+                         analysis = "sub_history_vascular_risk_false")
+
+  }
+
+  for (i in parkinson_risk) { #remove cov_bin_history_parkinson
+
+    ## analysis: sub_history_parkinson_risk_true ---------------------------------------
+
     df[nrow(df)+1,] <- c(cohort = c,
                          exposure = exposure,
                          outcome = i,
                          ipw = ipw,
                          strata = strata,
                          covariate_sex = covariate_sex,
                          covariate_age = covariate_age,
-                         covariate_other = gsub("cov_bin_history_parkinson;","",all_covars),
+                         covariate_other = gsub("cov_bin_history_parkinson;","",all_covars), #cov_bin_history_parkison_risk #cov_bin_history_parkinson
+                         #gsub("cov_bin_history_any_dementia;","",all_covars),
```

Comment:
Please can you remove any comments containing old code so that this is easier to read?

Reply:
Done.